schemas/compat-data.schema.json

```
@@ -147,6 +147,24 @@
           "pattern": "^https:\/\/developer\\.mozilla\\.org\/docs\/",
           "description": "A URL that points to an MDN reference page documenting the feature. The URL should be language-agnostic."
         },
+        "spec_url": {
+          "anyOf": [
+            {
+              "type": "string",
+              "format": "uri",
+              "pattern": "^http(s)?:\/\/[^#]+#.+"
+            },
+            {
+              "type": "array",
```

We should add that arrays require at least 2 items, so that people use the simple string when there is just one URL. This would be consistent with what https://github.com/mdn/browser-compat-data/pull/3090 wants to do.
```suggestion
              "type": "array",
              "minItems": 2,
```

---

> We should add that arrays require at least 2 items, so that people use the simple string when there is just one URL. This would be consistent with what #3090 wants to do.

OK, will make that change this weekend